ElectronClient/app/gui/NoteText.jsx

```diff
@@ -426,6 +433,13 @@ class NoteTextComponent extends React.Component {
 
 		this.setState(newState);
 
+		if (!this.props.newNote) {
```

Comment:
Had to add this check. If not, was going into an infinite loop where state was getting updated repeatedly. 

Since I'm updating the state, the `componentWillReceiveProps` was getting triggered again, where `nextProps.newNote` was still true, causing `reloadNote` to trigger again and again.

Not very happy with this piece of code, but not entirely sure how to approach the problem.

Reply:
That component was not very well designed to begin with since I was a bit new to React when I wrote it. The proper solution probably is to refactor and not use componentWillReceiveProps, which is now deprecated, but I'll check and see if I can think of a simpler solution.

Reply:
For this, let's leave it as it is for now, and I'm going to add your comment above the line.